README .R

```diff
@@ -0,0 +1,28 @@
+
+library(dplyr)
+library(readr)
+library(stringr)
+library(sbtools)
+library(whisker)
+
+source("1_fetch/src/fetch_data.R")
+source("2_process/src/process_data.R") 
+source("3_visualize/src/visualize_data.R")
+
+project_output_dir <- '1_fetch/out'
+mendota_file <- file.path(project_output_dir, 'model_RMSEs.csv')
+
+fetch_data(mendota_file)
```

Comment:
Great, I think this use of `fetch_data(mendota_file)` makes sense.

In the interest of keeping code simple, you have the option (not required, but perhaps more elegant) of combining lines 12-13 into one line. I'd do it like this:
```r
mendota_file <- '1_fetch/out/model_RMSEs.csv'
```

In an upcoming issue, I think you'll probably end up simplifying things even further, but that can wait until we have the right justification for it (the specific requirements of the **scipiper** framework)

Reply:
this comment still stands (in review number 2)